examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -96,23 +97,43 @@ def center_cmap(cmap, vmin, vmax):
 freqs = np.arange(2, 36, 1)  # frequencies from 2-35Hz
 n_cycles = freqs  # use constant t/f resolution
 vmin, vmax = -1, 1.5  # set min and max ERDS values in plot
+baseline = [-1, 0]  # baseline interval (in s)
 cmap = center_cmap(plt.cm.RdBu, vmin, vmax)  # zero maps to white
+kwargs = dict(n_permutations=100, step_down_p=0.05, seed=1)  # for cluster test
 
 for event in event_ids:
-    power = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
-                           use_fft=True, return_itc=False, decim=2)
-    power.crop(tmin, tmax)
-
-    fig, ax = plt.subplots(1, 4, figsize=(12, 4),
-                           gridspec_kw={"width_ratios": [10, 10, 10, 1]})
-    for i in range(3):
-        power.plot([i], baseline=[-1, 0], mode="percent", vmin=vmin, vmax=vmax,
-                   cmap=(cmap, False), axes=ax[i], colorbar=False, show=False)
-        ax[i].set_title(epochs.ch_names[i], fontsize=10)
-        ax[i].axvline(0, linewidth=1, color="black", linestyle=":")  # event
-        if i > 0:
-            ax[i].set_ylabel("")
-            ax[i].set_yticklabels("")
-    fig.colorbar(ax[0].collections[0], cax=ax[-1])
+    tfr = tfr_multitaper(epochs[event], freqs=freqs, n_cycles=n_cycles,
+                         use_fft=True, return_itc=False, average=False,
+                         decim=2)
+    tfr.crop(tmin, tmax)
+    tfr.apply_baseline(baseline, mode="percent")
+
+    fig, axs = plt.subplots(1, 4, figsize=(12, 4),
+                            gridspec_kw={"width_ratios": [10, 10, 10, 1]})
+    for ch, ax in enumerate(axs[:-1]):  # for each channel
+        mask = np.full(tfr.data.shape[-2:], False)  # initially mask all values
+
+        # positive clusters
+        _, c1, p1, _ = pcluster_test(tfr.data[:, ch, ...], tail=1, **kwargs)
+        # negative clusters
+        _, c2, p2, _ = pcluster_test(tfr.data[:, ch, ...], tail=-1, **kwargs)
+
+        # note that we keep clusters with p <= 0.05 from the combined clusters
+        # of two independent tests; in this example, we do not correct for
+        # these two comparisons
+        c = np.stack(c1 + c2, axis=2)  # combined clusters
+        p = np.concatenate((p1, p2))  # combined p-values
+        mask = np.logical_or.reduce(c[..., p <= 0.05], axis=-1)
```

Comment:
What black magic is this!

Reply:
You told me to vectorize it, I never said this would be more readable 😉.

Reply:
Same as `mask = c[..., p <= 0.05].any(-1)`?

Reply:
Yep, the same. I guess this is slightly more readable?

Reply:
if you want more readability you can do `.any(axis=-1)`. But in general I've seen people use `any` rather than `logical_or.reduce` (first time I've seen the latter)

Reply:
OK, `.any(axis=-1)` it is then!